.github/workflows/ci_test-tpu.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.7]
-        xla-version: [1.6, 1.7]
+        xla-version: [1.7, 1.8]
```

Comment:
Should we have [1.6, 1.8] as the minimum supported and latest?

Reply:
well good question what min do we want to support for XLA?
@PyTorchLightning/core-contributors

Reply:
Updated